fix dev_rules/hello.yml

to pass actions generate.yml

only change hello.yml and do not add new file for repository.
do not change other files.

error code



```
Run sg test --skip-snapshot-tests
Running 1 tests
Error: test failed. 0 passed; 1 failed;
Help: You can use ast-grep playground to debug your rules and test cases.
See also: https://ast-grep.github.io/playground.html


----------- Case Details -----------
[Missing] Expect rule hello to report issues, but none found in:

  //  Hello

FAIL hello  ..M

Error: Process completed with exit code 3.
```

--- a/dev_rules/hello.yml
+++ b/dev_rules/hello.yml
@@ -4,4 +4,4 @@ severity:  error
 language: systemverilog
 rule:
   kind: comment
-  regex: "// Hello"
+  regex: "//\\s*Hello"

